lib/shopify_api/resources/usage_charge.rb

```diff
@@ -0,0 +1,13 @@
+module ShopifyAPI
+  class UsageCharge < Base
+    init_prefix :recurring_application_charge
+
+    class << self
+      def index(recurring_application_charge_id)
+        UsageCharge.find(:all, :params => { :recurring_application_charge_id => recurring_application_charge_id })
+      rescue ActiveResource::ForbiddenAccess
```

Comment:
why are you catching this - if you let it bubble up doesn't the client get a 403 and wouldn't that be preferred?